vms/platformvm/blocks/executor/helpers_test.go

```diff
@@ -269,11 +270,13 @@ func defaultState(
 	rewards reward.Calculator,
 ) state.State {
 	genesisBytes := buildGenesisTest(ctx)
+	execCfg, _ := execconfig.GetExecutionConfig([]byte(`{}`))
 	state, err := state.New(
 		db,
 		genesisBytes,
 		prometheus.NewRegistry(),
 		cfg,
+		execCfg,
```

Comment:
I'd move trackChecksum parameter into the exec config. Similarly to other execConfig it can b set via a flag.